react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { drop: '', current: '' };
+    this.handleDropDown = this.handleDropDown.bind(this);
+    this.itemList = this.itemList.bind(this);
+    this.handleItemsClicked = this.handleItemsClicked.bind(this);
+  }
+
+  handleDropDown(e) {
+    const key = e.target.textContent;
+    this.setState({ current: this.state.drop });
+    this.setState({ drop: key });
```

Comment:
I would do this in one setState call, you just need an object with both properties and values